tests/test_api_jws.py

```diff
@@ -1,4 +1,5 @@
 import json
+from collections import OrderedDict
```

Comment:
can we use simple dict instead of orderedDict here?

Reply:
Thanks, I changed it in cf57f33aac5d1c2076abb14ecffdf3967739c5a6